service/certconfig/v2/resources/vaultcrt/desired_test.go

```diff
@@ -36,7 +36,7 @@ func Test_Resource_VaultCrt_GetDesiredState(t *testing.T) {
 				ObjectMeta: apismetav1.ObjectMeta{
 					Name: "foobar-api",
 					Annotations: map[string]string{
-						ConfigHashAnnotation:           "15d0a106b8d5f7f78dc37e73931619108322bcd7",
+						ConfigHashAnnotation:           "394f594f5cf6a2deb9abc6f0e322d887557d4a8e",
```

Comment:
The hashes change now and cause the certs to be renewed because the cert config structure changes. On the bright side this PR is about disable regeneration for service account certs. :D